upa/src/sdk/client.ts

```diff
@@ -139,3 +142,104 @@ export class UpaClient {
     return this.upaInstance.verifier.registerVK(vk.solidity());
   }
 }
+
+export type UnsignedOffChainSubmissionRequest = {
```

Comment:
To note, this conflicts a bit with what's happening in https://github.com/NebraZKP/upa/pull/49.

Let's discuss and coordinate.

Reply:
(We'll probably want a proper class with `from_json` initializers to thoroughly check the serialized data).  Let's combine with the above.  Maybe we can merge branches.

Reply:
turned into classes